src/applications/caregivers/tests/e2e/1010cg.cypress.spec.js

```diff
@@ -3,102 +3,62 @@ import testForm from 'platform/testing/e2e/cypress/support/form-tester';
-  'Signed by the Veteran’s legal representative on behalf of the Veteran.',
-  'I certify that I give consent to the individual(s) named in this application to perform personal care services for me (or if the Veteran’s Representative, the Veteran) upon being approved as a Primary and/or Secondary Family Caregiver(s) in the Program of Comprehensive Assistance for Family Caregivers.',
-];
-export const mockPrimaryCaregiverContent = [
-  'I certify that I am at least 18 years of age.',
-  'I certify that either: (1) I am a member of the Veteran\u2019s family (including a parent, spouse, a son or daughter, a step-family member, or an extended family member) OR (2) I am not a member of the Veteran\u2019s family, and I reside with the Veteran full-time or will do so upon designation as the Veteran\u2019s Primary Family Caregiver.',
-  'I agree to perform personal care services as the Primary Family Caregiver for the Veteran named on this application.',
-  'I understand that the Veteran or the Veteran\u2019s surrogate may request my discharge from the Program of Comprehensive Assistance for Family Caregivers (PCAFC) at any time and that my designation as a Primary Family Caregiver may be revoked or I may be discharged from PCAFC by the Secretary of Veterans Affairs (or designee) as set forth in 38 CFR 71.45.',
-  'I understand that participation in the PCAFC does not create an employment relationship between me and the Department of Veterans Affairs.',
-];
-export const mockSecondaryCaregiverContent = [
-  'I certify that I am at least 18 years of age.',
-  'I certify that either: (1) I am a member of the Veteran\u2019s family (including a parent, spouse, a son or daughter, a step-family member, or an extended family member) OR (2) I am not a member of the Veteran\u2019s family, and I reside with the Veteran full-time or will do so upon designation as the Veteran\u2019s Secondary Family Caregiver.',
-  'I agree to perform personal care services as the Secondary Family Caregiver for the Veteran named on this application.',
-  'I understand that the Veteran or the Veteran\u2019s surrogate may request my discharge from the Program of Comprehensive Assistance for Family Caregivers (PCAFC) at any time and that my designation as a Secondary Family Caregiver may be revoked or I may be discharged from PCAFC by the Secretary of Veterans Affairs (or designee) as set forth in 38 CFR 71.45.',
-  'I understand that participation in the PCAFC does not create an employment relationship between me and the Department of Veterans Affairs.',
-];
-
-const checkContent = (partyLabel, content, mockContent) => {
-  content.forEach((contentItem, idx) => {
-    cy.get(`[data-testid="${partyLabel}"]`)
-      .contains(contentItem, { matchCase: true })
-      .should(signatureParagraph =>
-        expect(signatureParagraph[0].innerText).to.eq(mockContent[idx]),
-      );
-  });
-};
-
-const signAsParty = (partyLabel, signature) => {
```

Comment:
Moved this to the utils file and renamed to `fillStatementOfTruthPattern` to match the other helper methods.